### PR TITLE
Spacehub PR

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
### In this milestone We @Santosh-Konappanavar, @ritikarawat220 

- we work on the actual live data from the SpaceX API. we build a web application for a company that 
provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
- Used React components, Used React props, Used React Router, Connected React and Redux
- Write unit tests with React Testing Library
- Used styles in a React app
- Used React hooks
- Use store, actions and reducers in React
- Performed a code review for a team member

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import Myprofile from './Routes/myprofile';
+
+import { fetchMissions } from './redux/mission/missionSlice';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
```
Comment (emmiiorji) [inline src/App.js:17]: You have done great dispatching the fetchMissions action. However, it's required that the action for fetching missions be dispatched only when the user navigates to the missions page (and only once). Kindly dispatch the action within the missions component in order to achieve this.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+.cancel-btn,
+.reserve-Btn {
+  background-color: #0290ff;
+  border: none;
+  padding: 0.5rem;
+  color: white;
+  cursor: pointer;
+  border-radius: 5px;
+}
+
```
Comment (emmiiorji) [inline src/App.css:113]: Kindly adjust the responsiveness on the rockets page and align all the rockets rendered. This is the display on 1510px screen

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Santosh-Konappanavar and @ritikarawat220 , 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Good job with following the design
- Tests are passed
- Good state management with redux 
- Great team work


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hello Team🙋‍♂️🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**